extension/background/intentRunner.js

```diff
@@ -146,7 +146,10 @@ export class IntentContext {
 
   async createTabGoogleLucky(query, options = {}) {
     const searchUrl = searching.googleSearchUrl(query, true);
-    const tab = await this.createTab({ url: searchUrl });
+    const tab =
+      !!options.youtubeTabId && options.youtubeTabId > -1
```

Comment:
There's nothing YouTube-specific to this code, so you could just make this `openInTabId` instead of `youtubeTabId`